tsconfig.json

```diff
@@ -10,6 +10,9 @@
     "inlineSourceMap": false,
     "inlineSources": false,
     "resolveJsonModule": true,
-    "useUnknownInCatchVariables": false
+    "useUnknownInCatchVariables": false,
+    "lib": [
+      "dom"
+    ],
```

Comment:
Test failed because of this. What I can add "esnext" or "es2019" or more ?

Reply:
You can add `es2020`